internal/build/ocpbuild/maker.go

```diff
@@ -138,8 +138,8 @@ func (m *maker) MakeBuildTemplate(
 		ObjectMeta: metav1.ObjectMeta{
 			GenerateName: mld.Name + "-build-",
 			Namespace:    mld.Namespace,
-			Labels:       buildutils.GetBuildLabels(mld, BuildType),
-			Annotations:  buildutils.GetBuildAnnotations(sourceConfigHash),
+			Labels:       ocpbuildutils.GetOCPBuildLabels(mld, BuildType),
+			Annotations:  ocpbuildutils.GetOCPBuildAnnotations(sourceConfigHash),
```

Comment:
Do we also need to add `OCP` to the function names based on the fact there are in the `ocpbuildutils` package?